ramls/oai-pmh-view.raml

```diff
@@ -49,4 +50,25 @@ resourceTypes:
           type: boolean
           required: false
           default: true
-
+  /instances:
+    displayName: Stream API to get instances with items and holdings for OAI-PMH
+    post:
+      description: Stream instances view data for oai-pmh
+      is:
+        [validate]
+      body:
+        application/json:
+          description: List of instances Ids
+          type: string[]
+      queryParameters:
+        skipSuppressedFromDiscoveryRecords:
+          description: skip discovery suppressed records
+          type: boolean
+          required: false
+          default: true
```

Comment:
There should be one get, one post

Reply:
What I'm trying to say is that the POST in this case behaves as a GET method - used to read records by some criteria, right?

Reply:
Yes, it's behaving like get. IT's made post because it accepts list of ids in the body.

Reply:
> This suggests that you're going to supply filter params as a POST body? But in a fact it is GET method?

Do you mean that despite the fact that this is a POST entry it uses "query parameter"? Of cource we can  pass it in body's payload. But for sake of simplyfiing payload processing we use single query parameter here. Does it make sence?

Reply:
> Yes, it's behaving like get. IT's made post because it accepts list of ids in the body.
If it was agreed with an SA I'm fine as well.

> But for sake of simplyfiing payload processing we use single query parameter here. Does it make sence?

I'd prefer specifying everything as either query params or as payload.

Reply:
As for parameter passing here, I removed single query parameter and moved it into body payload. Because there is particular data type was introduced in RAML, which carry all passed data ("instances ids" and "suppressed from discovery" flag). Thank you for raising this qurestion.